portality/app.py

```diff
@@ -237,7 +237,7 @@ def doi_url(doi):
     :return: the HTML link
     """
     tendot = doi[doi.find('10.'):]
-    return "<a href='https://doi.org/{0}'>{0}</a>".format(tendot)
+    return "https://doi.org/{0}".format(tendot)
```

Comment:
This is the same place where we spotted that issue yesterday around ascii characters.  If we're changing this line now, maybe worth checking for that issue and fixing it at the same time.

Reply:
Could you please remind me what issue that is about? Do we have an issue open?

Reply:
We talked about this out of band, but for reference it's this issue: https://github.com/DOAJ/doajPM/issues/2023

Reply:
Actually, on reflection, I think we need to do a bit more here.  As I mentioned in the other PR I just reviewed, there is a function `portality.lib.normalise.normalise_doi` and we should probably use that inside `doi_url`, so that we have a normalised DOI going on to the `https://doi.org/` prefix.